packages/opentelemetry-sdk-trace-base/src/export/SimpleSpanProcessor.ts

```diff
@@ -38,16 +38,16 @@ import { Resource } from '@opentelemetry/resources';
  */
 export class SimpleSpanProcessor implements SpanProcessor {
   private _shutdownOnce: BindOnceFuture<void>;
-  private _unresolvedExports: Set<Promise<void>>;
+  private _pendingExports: Set<Promise<void>>;
```

Comment:
unresolved seemed to be referring specifically to the resource so I renamed to pending to be more generic